### PR TITLE
Add double mark for compiler check related with issue #4835

### PR DESCRIPTION
**Pull request by eliakorkmaz**
### Issue Link :link:
https://github.com/danielgindi/Charts/issues/4835

in Xcode14 there is an compiler check error because of ambiguity of some math arithmetics, so this pr aim to fix it  

### Goals :soccer:
Because of this compiler error, i aim to fix this problem for xcode14-beta versions

### Implementation Details :construction:
There are no architectural changes, I just put Double casting into magic number as `1.0`

### CODE REVIEW

**Review by MichalMartinu — APPROVED** [review]
👍

**Comment by eliakorkmaz** [comment]
thank you @MichalMartinu , this pr is still open because of 1 workflow awaiting approval

**Comment by eliakorkmaz** [comment]
hey @MichalMartinu , a maintainer with write access may need to approve any workflow runs. 

Can you give approve to workflow run?

**Comment by lchamp** [comment]
Hi @danielgindi, @pmairoldi, can you approve this PR to provide support for Xcode14(beta)?
Thanks in advance!

**Comment by pmairoldi** [comment]
We usually do not support betas as things can change all the time. When Xcode 14 is GM we will release something if need be. Will keep this open for now.

**Comment by pmairoldi** [comment]
In beta 6 I dont get this error anymore. Closing

**Comment by pmairoldi** [comment]
Actually never mind. The library compiles fine but this is needed to run in an app. Kinda weird.

**Comment by pmairoldi** [comment]
Thanks for the contribution!

**Comment by pmairoldi** [comment]
I will wait until the Xcode 14 gm and if everything is still fine I’ll release a new version.

**Pull request merged**
